web/src/admin/AdminPoster.js

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import { withRouter, Link } from "react-router-dom";
+import * as MemberBackend from "../backend/MemberBackend.js";
+import * as PosterBackend from "../backend/PosterBackend"
+import * as Setting from "../Setting";
+import PageColumn from "../main/PageColumn";
+import Avatar from "../Avatar";
+import Collapse, { Panel } from "rc-collapse";
+import Select2 from "react-select2-wrapper";
+import i18next from "i18next";
+
+class AdminMember extends React.Component {
+    constructor(props) {
+        super(props);
+        this.adver=React.createRef();
+        this.links=React.createRef();
+        this.p_link=React.createRef();
+        this.state = {
+            classes: props,
+            message: "",
+            errorMessage: "",
+            form: {},
+            //event: props.match.params.event,
+            membersNum: 0,
+            members: [],
+            width: "",
+            memberId: props.match.params.memberId,
+            member: [],
+            event: "basic",
+            page: 1,
+            limit: 100,
+            minPage: 1,
+            maxPage: -1,
+            showSearch: false,
+            loading: true,
+            un: "", // username search
+            cs: 1, // created time sort, default: Asc
+            us: 0, // username sort, default: None
+            Status_LIST: [
+                { label: "Normal", value: 0 },
+                { label: "Mute", value: 1 },
+                { label: "Forbidden", value: 2 },
+            ],
+            Management_LIST: [
+                { label: "Basic Info", value: "basic" },
+                { label: "Personal Info", value: "personal" },
+            ],
+        };
+        const params = new URLSearchParams(this.props.location.search);
+        this.state.p = params.get("p");
+        if (this.state.p === null) {
+            this.state.page = 1;
+        } else {
+            this.state.page = parseInt(this.state.p);
+        }
+
+        this.state.url = `/admin/poster`;
+    }
+
+    componentDidMount() {
+    }
+
+    updateposter() {
+        PosterBackend.updateposter_info(this.state.memberId, this.state.form).then(
```

Comment:
Why memberId?

Reply:
> 为什么是memberId？
Use member id to determine whether it can be changed

Reply:
> 为什么是memberId？

Use member id to determine whether it can be changed